### PR TITLE
Updated prerequisite to AWS S3 compatible bucket for log storage.

### PR DESCRIPTION
**Pull request by mikesmith0414**
Updated prerequisite to AWS S3 compatible bucket for log storage.

Version(s): 4.10+

Issue: https://github.com/openshift/openshift-docs/issues/50537

https://50531--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-loki.html#logging-loki-deploy_cluster-logging-loki

QE needed

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

**Comment by mjpytlak** [comment]
@mikesmith0414 @stevsmit The preview appears to be out of date. Can you force push to kick off Netlify again? I just need to verify the changes in the published output.

**Comment by mikesmith0414** [comment]
fixed @mjpytlak

File: modules/cluster-logging-loki-deploy.adoc [hunk 1]
```diff
@@ -8,7 +8,7 @@ You can use the {product-title} web console to deploy the LokiStack.
 .Prerequisites
 
 * {logging-title-uc} Operator 5.5 and later
-* AWS S3 bucket for log storage
+* AWS S3 compatible bucket for log storage
```
Comment (jeana-redhat) [inline modules/cluster-logging-loki-deploy.adoc:11]: **NOTE:** GitHub will make a button for this comment that says something like "accept suggestion". Don't click it, it does weird stuff that's harder to get through than just editing normally :slightly_smiling_face: 

Need a hyphen for the compound adjective here:
```suggestion
* AWS S3-compatible bucket for log storage
```

**Review by jeana-redhat — COMMENTED** [review]
One style tweak for this before merging please

**Comment by bergerhoffer** [comment]
The `branch/enterprise-4.13` label has been added to this PR.

This is because your PR targets the `main` branch and is labeled for enterprise-4.12. And any PR going into `main` must also target the latest version branch (enterprise-4.13).

If the update in your PR does NOT apply to version 4.13 onward, please re-target this PR to go directly into the appropriate version branch or branches (enterprise-4.x) instead of main.

**Review by anpingli — COMMENTED** [review]
/lgtm

**Pull request closed without merging**